packages/oasgraph/lib/types/graphql.d.ts

```diff
@@ -10,7 +10,7 @@ declare type Arg = {
 export declare type Args = {
     [key: string]: Arg;
 };
-export declare type ResolveFunction = (root: Object, args: Object, ctx: Object) => Promise<any> | any;
+export declare type ResolveFunction = (root: Object, args: Object, ctx: Object, info: Object) => Promise<any> | any;
```

Comment:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html